orte/mca/schizo/base/schizo_base_stubs.c

```diff
@@ -46,6 +46,11 @@ int orte_schizo_base_parse_env(char *personality,
     int rc;
     orte_schizo_base_active_module_t *mod;
 
+    if (NULL == personality) {
+        opal_output(0, "NULL PERSONALITY");
```

Comment:
Updated to just error_log it and return the error. Since we return an error if no module can support that operation, this is no worse than the normal error path.